### PR TITLE
STYLE: Use ITK clang-format style

### PR DESCRIPTION
**Pull request by thewtex**

### CODE REVIEW

File: {{cookiecutter.project_name}}/test/itk{{cookiecutter.filter_name}}Test.cxx [hunk 1]
```diff
@@ -16,81 +16,82 @@
  *
  *=========================================================================*/
 
-#include "itk{{cookiecutter.filter_name}}.h"
+#include "itk{{ cookiecutter.filter_name }}.h"
```
Comment (dzenanz) [inline {{cookiecutter.project_name}}/test/itk{{cookiecutter.filter_name}}Test.cxx]: Will spaces here be a problem?

File: {{cookiecutter.project_name}}/test/itk{{cookiecutter.filter_name}}Test.cxx [hunk 2]
```diff
@@ -16,81 +16,82 @@
...
-      }
+    }
     std::cout << " " << processObject->GetProgress();
   }
 };
-}
+} // namespace
 
-int itk{{cookiecutter.filter_name}}Test( int argc, char * argv[] )
+int itk{{ cookiecutter.filter_name }}Test(int argc, char * argv[])
```
Comment (dzenanz) [inline {{cookiecutter.project_name}}/test/itk{{cookiecutter.filter_name}}Test.cxx]: Are spaces a problem?

File: {{cookiecutter.project_name}}/test/itk{{cookiecutter.filter_name}}Test.cxx (hunk 1 above)
Comment (thewtex) [reply {{cookiecutter.project_name}}/test/itk{{cookiecutter.filter_name}}Test.cxx]: The templating system (jinja2) supports the spaces, and they also help readability.

**Comment by thewtex** [comment]
> Did you just run clang-format on these files as is, or did you replace it with token names, ran the style, then transferred changes to the original files?

I ran clang-format on the files, but I had to manually go back and fix them because clang-format did not understand the template macro syntax.

**Pull request merged**